cmd/vic-machine/common/dns.go

```diff
@@ -25,15 +25,15 @@ import (
 
 // general dns
 type DNS struct {
-	dns   cli.StringSlice `arg:"dns-server"`
+	Dns   cli.StringSlice `arg:"dns-server"`
 	IsSet bool
 }
 
 func (d *DNS) DNSFlags(hidden bool) []cli.Flag {
 	return []cli.Flag{
 		cli.StringSliceFlag{
 			Name:   "dns-server",
-			Value:  &d.dns,
+			Value:  &d.Dns,
 			Usage:  "DNS server for the client, public, and management networks. Defaults to 8.8.8.8 and 8.8.4.4 when VCH uses static IP",
```

Comment:
Should the usage specify how to enter multiple values (comma/space separated, etc)?

Reply:
To specify multiple servers I believe you specify the argument multiple times (`--dns 8.8.8.8 --dns 8.8.4.4`), but that's not clear from the current usage string.

Reply:
This is from CLI and for example if we have 3 DNS servers, we just give it 3 `dns-server` args:
```--dns-server=1.1.1.1 --dns-server=2.2.2.2 --dns-server=3.3.3.3```